packages/enzyme-adapter-react-13/src/ReactThirteenAdapter.js

```diff
@@ -280,6 +285,13 @@ class ReactThirteenAdapter extends EnzymeAdapter {
   createElement(...args) {
     return React.createElement(...args);
   }
+
+  wrapWithWrappingComponent(node, options) {
+    return {
+      RootFinder,
```

Comment:
how come this one adapter doesn't need to `...this` in its new renderer?

Also, if it's not going to be supported in react 13 yet (since the tests don't cover it), let's have this method instead throw an explicit error?

Reply:
Added a test for throwing an error in react 13!